rasa/core/run.py

```diff
@@ -34,6 +34,12 @@ def create_http_input_channels(
         all_credentials = {}
 
     if channel:
+        if len(all_credentials) > 1:
+            logger.warning(
```

Comment:
I'd prefer to use `info` since it's the desired behavior and a warning regarding some potential problem. What do you think?
```suggestion
            logger.info(
```